src/chrome/content/rules/GitHub.xml

```diff
@@ -1,16 +1,31 @@
 <!--
-	Other GitHub rulesets:
+	403 status code (can't find valid test URLs, therefore not including):
+		- assets.github.com
+
+	404 status code (can't find valid test URLs, therefore not including):
+		- live.github.com
 
+	Other GitHub rulesets:
 		- Choosealicense.com.xml
 		- Github-Pages.xml
 		- Guag.es.xml
 		- Speaker_Deck.com.xml
 
+	Redirects:
+		- codeload.github.com → github.com
+		- learn.github.com → services.github.com
+		- raw-origin.github.com → github.com
+		- teach.github.com → services.github.com
+		- training.github.com → services.github.com
+
+	Secure connection failed:
+		- ssh.github.com
 -->
```

Comment:
Since we already have github.com proloaded, I think it's fine to remove these. Maybe you could just leave a comment to state that it's HSTS preloaded and thus doesn't need to be in the ruleset.

Reply:
Done.

Reply:
Actually you don't need to mention the subdomains at all, as includeSubDomains is one of the requirements to get preloaded

Reply:
The newest commit should have fixed that.